cluster/addons/addon-manager/kube-addons.sh

```diff
@@ -194,9 +196,9 @@ function is_leader() {
     log INFO "Leader election disabled."
     return 0;
   fi
-  KUBE_CONTROLLER_MANAGER_LEADER=`${KUBECTL} -n kube-system get ep kube-controller-manager \
+  KUBE_CONTROLLER_MANAGER_LEADER=$(${KUBECTL} -n kube-system get ep kube-controller-manager \
     -o go-template=$'{{index .metadata.annotations "control-plane.alpha.kubernetes.io/leader"}}' \
-    | sed 's/^.*"holderIdentity":"\([^"]*\)".*/\1/' | awk -F'_' '{print $1}'`
+    | sed 's/^.*"holderIdentity":"\([^"]*\)".*/\1/' | awk -F'_' '{print $1}')
```

Comment:
change reason:
```
In ./cluster/addons/addon-manager/kube-addons.sh line 197:
  KUBE_CONTROLLER_MANAGER_LEADER=`${KUBECTL} -n kube-system get ep kube-controller-manager \
                                 ^-- SC2006: Use $(...) notation instead of legacy backticked `...`.
```